e2e/support/timings.json

```diff
@@ -1,1168 +1,1324 @@
 {
   "durations": [
     {
-      "duration": 45000,
-      "spec": "../test/scenarios/actions/actions-in-object-detail-view.cy.spec.js"
+      "spec": "e2e/test/scenarios/actions/actions-in-object-detail-view.cy.spec.js",
+      "duration": 42945
     },
     {
-      "duration": 256000,
-      "spec": "../test/scenarios/actions/actions-on-dashboards.cy.spec.js"
+      "spec": "e2e/test/scenarios/actions/actions-on-dashboards.cy.spec.js",
```

Comment:
are you sure this works without the relative path? I recall there being a gotcha there 🤔

Reply:
That's how cypress-split was generating them in my other PR. And I believe I tested it with a little demo app and it was working there